packages/app/src/metric-config/index.ts

```diff
@@ -21,21 +20,24 @@ export type DataScope = 'nl' | 'vr' | 'gm';
 const metricConfig = {
   nl,
   vr,
-  gm,
 } as const;
 
 export function getMetricConfig(
   scope: DataScope,
   metricName: string,
-  metricProperty = NO_METRIC_PROPERTY
+  metricProperty?: string
 ) {
   /**
    * Fall back to an empty object so we don't have to specify empty objects in
    * the config file for properties that do not need a config. Since all
    * root-level properties in the MetricConfig are optional, an empty object is
    * still a valid configuration.
    */
-  const config = get(metricConfig, [scope, metricName, metricProperty], {});
+  const config = get(
+    metricConfig,
+    [scope, metricName, metricProperty ?? ''],
```

Comment:
nit; I think this is a little cleaner, because then you don't use '' as a property.

```suggestion
    metricProperty ? [scope, metricName, metricProperty] : [scope, metricName],
```